code-name-taran/src/com/spaghettisoft/component/scrabble/Dictionary.java

```diff
@@ -0,0 +1,39 @@
+package com.spaghettisoft.component.scrabble;
+
+public class Dictionary {

```

Comment:
What you are trying to do is not a good idea. A better solution is to find a file with all english words and just check if a word is inside the file.